test/jdk/java/awt/Modal/PrintDialogsTest/PrintDialogsTest.java

```diff
@@ -25,21 +25,76 @@
 /*
  * @test
  * @bug 8055836 8057694 8055752
- * @summary Check if Print and Page Setup dialogs lock other windows;
+ * @summary Check if Print and Page Setup dialogs block other windows;
  *          check also correctness of modal behavior for other dialogs.
- *
- * @run applet/manual=yesno PrintDialogsTest.html
+ * @library /java/awt/regtesthelpers
+ * @run main/manual PrintDialogsTest
  */
 
 
-import java.applet.Applet;
-import java.awt.*;
+import java.awt.BorderLayout;
+import java.awt.Button;
+import java.awt.Checkbox;
+import java.awt.CheckboxGroup;
+import java.awt.Dialog;
+import java.awt.Frame;
+import java.awt.EventQueue;
+import java.awt.GridLayout;
+import java.awt.Label;
+import java.awt.Panel;
 
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 
 
-public class PrintDialogsTest extends Applet implements ActionListener {
+public class PrintDialogsTest extends Panel implements ActionListener {
+
+    static final String INSTRUCTIONS = """
+        This test is free format, which means there is no enforced or guided sequence.
+
+        Please select each of
+        (a) The dialog parent type.
+        (b) The dialog modality type
+        (c) The print dialog type (Print dialog or Page Setup dialog)
```

Comment:
Can we put a one line gap in the GUI between Dialog parent type options and next dialog modality type heading...Right now the type and the options of all 3 appear one after the other,,,maybe 1 line gap between each category would help 

![image](https://github.com/openjdk/jdk/assets/43534309/a6c0f3f2-def3-4648-a13c-6e720bf6350c)